pyproject.toml

```diff
@@ -54,16 +54,20 @@ entrypoints = "^0.3"
 importlib-metadata = {version = "^1.6.0", python = "<3.8"}
 dataclasses = {version = "^0.8", python = "~3.6"}
 
+
 [tool.poetry.dev-dependencies]
+tox = "^3.18"
 pytest = "^6.2"
 pytest-cov = "^2.8"
 pytest-mock = "^3.5"
-pre-commit = { version = "^2.6", python = "^3.6.1" }
-tox = "^3.18"
 pytest-sugar = "^0.9"
+pytest-clarity = "^1.0.1"
+pytest-github-actions-annotate-failures = "^0.1.5"
```

Comment:
should we move this to an extra? and make it optional?

Reply:
We could certainly. The code is a no-op unless `$GITHUB_RUNNER` is in the environment, but it will pull in a few more dependencies. I'm open to going either way.

Reply:
I'd either take that out of pyproject entirely and install in the action run, however that might mean we might have incorrect dependencies. Hence I am leaning more towards a new group (now that we have groups) rather than extras come to think of it :)